frameworks/elastic/universe/marathon.json.mustache

```diff
@@ -433,6 +433,15 @@
     {{#elasticsearch.xpack_security_audit_enabled}}
     "TASKCFG_ALL_XPACK_SECURITY_AUDIT_ENABLED": "{{elasticsearch.xpack_security_audit_enabled}}",
     {{/elasticsearch.xpack_security_audit_enabled}}
+    {{#elasticsearch.xpack_security_transport_ssl_client_authentication}}
+    "TASKCFG_ALL_XPACK_SECURITY_TRANSPORT_SSL_CLIENT_AUTHENTICATION": "{{elasticsearch.xpack_security_transport_ssl_client_authentication}}",
+    {{/elasticsearch.xpack_security_transport_ssl_client_authentication}}
+    {{#elasticsearch.xpack_security_transport_ssl_verification_mode}}
+    "TASKCFG_ALL_XPACK_SECURITY_TRANSPORT_SSL_VERIFICATION_MODE": "{{elasticsearch.xpack_security_transport_ssl_verification_mode}}",
+    {{/elasticsearch.xpack_security_transport_ssl_verification_mode}}
+    {{#elasticsearch.xpack_security_http_ssl_client_authentication}}
+    "TASKCFG_ALL_XPACK_SECURITY_HTTP_SSL_CLIENT_AUTHENTICATION": "{{elasticsearch.xpack_security_http_ssl_client_authentication}}",
+    {{/elasticsearch.xpack_security_http_ssl_client_authentication}}
```

Comment:
I think all settings will need to be passed unconditionally so that the default values are reflected in `elasticsearch.yml` right?